recipes/cryolobm/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "cryoloBM" %}
+{% set version = "1.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cryoloBM-{{ version }}.tar.gz
+  sha256: e7552fad5f1e7f8f70af53592e9d09e6dd0ebdec4731960562c57bf8f6f516e4
+
+build:
+  entry_points:
+    - cryolo_boxmanager_legacy.py = cryoloBM.boxmanager:run
+    - cryolo_boxmanager_tools.py = cryoloBM.boxmanager_tools:_main_
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib-base >=3.4.2,<3.5
+    - numpy >=1.16.0,<1.19.0
+    - pystardb >=0.3.0
+    - pandas ==1.1.4
```

Comment:
Is there a reason for the hard pin here? It would be better to relax the pin.

Reply:
Relaxed it. I think its not necessary anymore.